domain-models-runtime/src/main/java/org/folio/rest/RestVerticle.java

```diff
@@ -770,13 +781,7 @@ private boolean isStreamed(Annotation[] annotations) {
    *          - request's start time, using JVM's high-resolution time source, in nanoseconds
    */
   private void sendResponse(RoutingContext rc, AsyncResult<Response> v, long start, String tenantId) {
-    Response result = v.result();
-    if (v.failed()) {
-      Throwable exception = v.cause();
-      if (exception instanceof ResponseException) {
-        result = ((ResponseException) exception).getResponse();
-      }
-    }
+    Response result = getResponse(v);
```

Comment:
```suggestion
    Response response = getResponse(v);
```

Reply:
This name is already used for `HttpServerResponse response` a few lines below.

Reply:
Then maybe it could be called something like `responseFromResult`?

Reply:
Yes, done.